### ISSUE TITLE
keybase gui failing to launch on debian

### ISSUE THREAD

**Issue by mossy-nw** [keybase/client#18897]
Hello,

Since updating keybase this morning on a debian-9 machine, the keybase gui refuses do launch.  The drawer applet does not appear, and `run_keybase` from the terminal fails to start the gui.

`systemctl --user status keybase.gui` in a terminal gives the following:

~~~
● keybase.gui.service - Keybase GUI
   Loaded: loaded (/usr/lib/systemd/user/keybase.gui.service; static; vendor pre
   Active: failed (Result: signal) since Mon 2019-08-12 09:44:39 PDT; 5s ago
  Process: 3228 ExecStartPost=/usr/bin/env systemctl --user unset-environment KE
  Process: 3227 ExecStart=/opt/keybase/Keybase (code=killed, signal=TRAP)
 Main PID: 3227 (code=killed, signal=TRAP)
~~~

`keybase status` gives the following:
(all identifying machine or account info replaced with `user`)

~~~
Username:      user
Logged in:     yes

Device:
    name:      device
    ID:        3c84c06be25374cda8b1e1f29715a318
    status:    active

Session:
    is valid:  yes
    keys:      unlocked

Key status:
    stream:    not cached
    secret:    stored
    dev sig:   cached
    dev enc:   cached
    paper sig: not cached
    paper enc: not cached
    prompt:    show
    tsec:      not cached

KBFS:
    status:    running
    version:   4.3.0-20190812143918+f8de90a10f
    installed: 4.3.0-20190812143918+f8de90a10f
    log:       /home/user/.cache/keybase/keybase.kbfs.log
    mount:     /run/user/1001/keybase/kbfs

Service:
    status:    running
    version:   4.3.0-20190812143918+f8de90a10f
    log:       /home/user/.cache/keybase/keybase.service.log
    eklog:     /home/user/.cache/keybase/keybase.ek.log

Updater:
    log:       /home/user/.cache/keybase/keybase.updater.log

Platform Information:
    OS:        linux
    OS vers:   Linux 4.9.0-9-amd64 x86_64-Debian GNU/Linux 9.9 (stretch)
    Runtime:   go1.12.4
    Arch:      amd64

Client:
    version:   4.3.0-20190812143918+f8de90a10f

Desktop app:
    status:    not running
    version:   
    log:       /home/user/.cache/keybase/Keybase.app.log

Config path:   /home/user/.config/keybase/config.json
Default user:  user
Other users:   
Configured accounts:
    user (current, logged in)
Known DeviceEKs:
    deviceEphemeralKey-user-1-197.ek
    deviceEphemeralKey-user-1-198.ek
    deviceEphemeralKey-user-1-199.ek
    deviceEphemeralKey-user-1-200.ek
    deviceEphemeralKey-user-1-201.ek
    deviceEphemeralKey-user-1-202.ek
    deviceEphemeralKey-user-1-203.ek
    deviceEphemeralKey-user-1-204.ek
    deviceEphemeralKey-user-1-205.ek 
LocalDbStats:
Compactions
 Level |   Tables   |    Size(MB)   |    Time(sec)  |    Read(MB)   |   Write(MB)
-------+------------+---------------+---------------+---------------+---------------
   1   |         11 |      99.12279 |       3.62289 |     100.38365 |      99.19384
   2   |        187 |     502.35616 |       0.70048 |      20.23702 |      20.16597
-------+------------+---------------+---------------+---------------+---------------
 Total |        198 |     601.47895 |       4.32337 |     120.62067 |     119.35981

levelDbCleaner{cacheSize: 598, lastRun: 2019-08-12 08:58:56.894223352 -0700 PDT m=-3239.973558276, lastKey: [], running: false}
DbCleanerConfig{MaxSize: 2.1GB, HaltSize: 1.6GB, CleanInterval: 1h0m0s, CacheCapacity: 100000, MinCacheSize: 10000, SleepInterval: 50ms}
 
LocalChatDbStats:
Compactions
 Level |   Tables   |    Size(MB)   |    Time(sec)  |    Read(MB)   |   Write(MB)
-------+------------+---------------+---------------+---------------+---------------
   0   |          0 |       0.00000 |       0.55032 |       0.00000 |      19.53627
   1   |          2 |      12.24932 |       0.55139 |      44.95289 |      23.34563
-------+------------+---------------+---------------+---------------+---------------
 Total |          2 |      12.24932 |       1.10171 |      44.95289 |      42.88190

levelDbCleaner{cacheSize: 113, lastRun: 2019-08-12 08:58:56.896717732 -0700 PDT m=-3239.971063906, lastKey: [], running: false}
DbCleanerConfig{MaxSize: 2.1GB, HaltSize: 1.6GB, CleanInterval: 1h0m0s, CacheCapacity: 100000, MinCacheSize: 10000, SleepInterval: 50ms}
 
LocalBlockCacheDbStats:
Compactions
 Level |   Tables   |    Size(MB)   |    Time(sec)  |    Read(MB)   |   Write(MB)
-------+------------+---------------+---------------+---------------+---------------
   1   |          3 |      69.69138 |       0.00000 |       0.00000 |       0.00000
   2   |         22 |     971.78981 |       0.00000 |       0.00000 |       0.00000
   3   |         48 |    2503.37286 |       0.00000 |       0.00000 |       0.00000
-------+------------+---------------+---------------+---------------+---------------
 Total |         73 |    3544.85404 |       0.00000 |       0.00000 |       0.00000
 
LocalSyncCacheDbStats:
Compactions
 Level |   Tables   |    Size(MB)   |    Time(sec)  |    Read(MB)   |   Write(MB)
-------+------------+---------------+---------------+---------------+---------------
-------+------------+---------------+---------------+---------------+---------------
 Total |          0 |       0.00000 |       0.00000 |       0.00000 |       0.00000
 
CacheDirSizeInfo:
/home/user/.cache/keybase/attachments: 728kB, (13 files)
/home/user/.cache/keybase/avatars: 24MB, (333 files)
/home/user/.cache/keybase: 612MB, (346 files)

command-line client:
    keybase status [cid: 4, pid: 947, version: 4.3.0-20190812143918+f8de90a10f]

KBFS:
    /usr/bin/kbfsfuse -debug -log-to-file [cid: 2, pid: 3567, version: 4.3.0-20190812143918+f8de90a10f]
    Notification subscriptions: Session, Favorites, Paperkeys, Keyfamily, Service, Kbfsrequest, Reachability, Team, Chatkbfsedits
~~~

**Comment by xmaysonnave** [comment]
I confirm I have the same behaviour. The GUI do not want to start.
- Debian Buster Linux andaman 4.19.0-5-amd64 #1 SMP Debian 4.19.37-5+deb10u2 (2019-08-08) x86_64 GNU/Linux
- keybase version 4.3.0-20190812143918+f8de90a10f
Here is my log file :
-- Begin syslog -- 
Aug 13 09:03:39 andaman Keybase[2105]: [2105:0813/090339.535584:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'│
│Aug 13 09:03:39 andaman systemd[1389]: keybase.gui.service: Main process exited, code=killed, status=5/TRAP                                                                                                        │
│Aug 13 09:03:39 andaman systemd[1389]: keybase.gui.service: Failed with result 'signal'.                                                                                                                           │
│Aug 13 09:03:39 andaman kernel: [   88.115403] traps: Keybase[2105] trap int3 ip:55e20f7fe537 sp:7ffc121eb8c0 error:0 in Keybase[55e20d8db000+5023000]  
-- End Syslog --
Thanks

**Comment by Josyann** [comment]
Same here, but I tried:

`/opt/keybase/Keybase`

and I got:

```
[13193:0813/102905.098962:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/keybase/chrome-sandbox is owned by root and has mode 4755.
[1]    13193 trace trap  /opt/keybase/Keybase
```
As the message said, I chmod chrome-sandbox and now it's working. But links in chat are unclickable.

Hope this solution is helping :confused:

**Comment by heronhaye** [comment]
We should get a patch out later today for Debian. In the meantime, you can solve this by running `chmod u+s /opt/keybase/chrome-sandbox`.

**Comment by heronhaye** [comment]
Hi, this should be fixed if you upgrade to 4.3.1. Let me know if doesn't work, thanks.
